DependencyInjection/Configuration.php

```diff
@@ -213,6 +213,14 @@ public function getConfigTreeBuilder()
                                 continue;
                             }
 
+                            if (isset($template['inherits_containers'])) {
+                                $inherits = $template['inherits_containers'];
+                                if (!isset($templates[$inherits])) {
+                                    throw new InvalidConfigurationException(sprintf('Template "%s" cannot inherits containers from undefined template "%s"', $id, $inherits));
```

Comment:
Trust me, no "s" here. The verb here is "cannot" and it doesn't take an "s" either.